### DISCUSSION TITLE
Audio file became inaudible overtime (for some participants)

### DISCUSSION THREAD

**Discussion by zhangchi199133** [jspsych/jsPsych#1035] [Q&A]
Hi all. I am now conducting an experiment using jsPsych plugin `audio-button-response`. I recently found that while for most of the participants the sound from the audio file was all right, for some of the participants the sound was normal for the around first 1/3 of the participants, then it collapsed into inaudible noises. All sound files are preloaded. There were no warning from the browser console when the problem occured. Does anybody know what might cause this problem? Thanks a lot!

p.s. After some examination I suspect there might be something to do with Firefox. But I am not entirely sure whether this was the case and where was the bug.

Best regards,

Chi

**Comment by jodeleeuw** [comment]
Hi Chi,

I've never heard of this problem before. What format are your audio files in?

**Reply by zhangchi199133** [reply]
Hi Josh,

My audio files are all in mp3 files. The problem occurred to only a portion of the participants. The experiment only allows Chrome and Firefox. Those that reported the error were all Firefox users. I am not so sure whether the problem also occurred to Chrome users.

Best regards,

Chi

**Reply by jodeleeuw** [reply]
Are you using webaudio or html5 audio? (Did you set `use_webaudio: false` in `jsPsych.init()`?)

You could try switching from one to the other to see if that makes a difference. This is a totally new kind of problem to me, so I'm not sure what else to suggest!

**Reply by zhangchi199133** [reply]
Hi Josh,

I enabled `use_webaudio` in my script. I could not check the bug on my computer because somehow the problem won't come about in any of the browsers I use (and this was the reason I did not preemptively detect the problem before the online testing started). I will find someone who has this problem and let him/her test it on my behalf.

Have a nice weekend!

Chi
